src/FilterList.js

```diff
@@ -110,7 +111,11 @@ export let Label = _.flow(
       }}
     >
       <span {...props}>
-        {children || _.get([field, 'label'], fields) || field || ''}
+        {children ||
+          _.get([key, 'label'], fields) ||
```

Comment:
Why would we check key? Keys are arbitrary and not likely to be in the schema 🤔

Reply:
Our fields are not just the schema, it's schema + overrides where we specify labels by the key. Think of several facets for the same field (title contains/doesn't)

Reply:
We typically handle this with mapNodeToLabel. The problem with coding against keys is that they aren't guaranteed to be present since you can generally remove and re-add nodes. In your example, someone can remove title contains and does not contain and then add them back. FilterList generally blocks duplicate fields, but query builder does not. 

Also, having schemas have fields that are keys but not fields seems like a recipe for confusion.

Reply:
I see that makes sense now. I will reverse that and add mapNodeToLabel for filter buttons